lib/spreewald_support/mail_finder.rb

```diff
@@ -64,6 +64,9 @@ def header_representation(mail)
       header << "CC: #{mail.cc.join(', ')}\n" if mail.cc
       header << "BCC: #{mail.bcc.join(', ')}\n" if mail.bcc
       header << "Subject: #{mail.subject}\n"
+      header << "Attachments: #{Array(mail.attachments).collect(&:filename).sort.join(', ')}\n" if mail.attachments.any?
```

Comment:
Oben in Zeile 35 wird bereits etwas ähnliches gemacht.
```ruby
Array(mail.attachments).collect(&:"#{filename_method}").sort
```

Könntest du das bitte in eine eigene Methode extrahieren, die dann auch die `filename_method` bestimmt?

```ruby
def attachments(mail)
  filename_method = Rails::VERSION::MAJOR < 3 ? 'original_filename' : 'filename'
  Array(mail.attachments).collect(&:"#{filename_method}").sort
end
```

Reply:
Das is blöd, `Rails::VERSION::MAJOR` ist in den Specs nicht da, da wird nur das Gem getested ohne Rails. 
Weis nicht, wie des am besten fix.
Am einfachsten wärs einfach hier nur `filename` zu nehmen und damit Rails 2 support zu droppen. Würd mich wundern, wenn das noch jemand brauchen würd. Was meinst du?

Reply:
Theoretisch könnte man mit Rails LTS noch Version 2 haben. Man könnte auch spreewald ganz ohne Rails verwenden, dann funktioniert der MailFinder aktuell nicht. action_mailer könnte man auch ohne Rails im Gemfile haben (macht vermutlich keiner).

Hässlich, aber würde funktionieren:
```
filename_method = (defined?(Rails::VERSION::MAJOR) && Rails::VERSION::MAJOR < 3) ? 'original_filename' : 'filename'
```

Reply:
Die Methode kommt aus dem `mail` gem.
Ich kann nicht ganz nachvolziehen, wann sich da der Name der Methode gändert hat. 
Mit Rails 3 ging das mail von 1.5 auf 2.0, ich denk das wird gewesen sein, aber sicher bin ich mir nicht. 
Hab auch wenig Möglichkeit das zu testen.

Ich würd echt einfach den Support da droppen, wer noch aktiv an seiner Rails 2 App arbeitet und eine neue spreewald Version will, soll einfach sein Rails updaten. :whatyear:  Würd mich auch bisschen wundern wenn das aktuelle spreewald da sonst noch kompatibel ist. 
Wir haben ja auch nur noch Tests für Rails 7. :shrug: